src/MessageHandler/User/CreateOAuthUserHandler.php

```diff
@@ -35,5 +41,11 @@ public function __invoke(CreateOAuthUser $message): void
         $user->confirmEmail($confirmToken);
 
         $this->users->add($user);
+
+        $this->messageBus->dispatch(
+            (new Envelope(
+                new GenerateApiToken($id->toString(), 'default'))
+            )->with(new DispatchAfterCurrentBusStamp())
+        );
```

Comment:
nope, maybe let the user decide for himself when he wants to generate the api token?